bravado/testing/response_mocks.py

```diff
@@ -0,0 +1,81 @@
+# -*- coding: utf-8 -*-
+from bravado_core.response import IncomingResponse
+
+from bravado.exception import BravadoTimeoutError
+from bravado.http_future import FALLBACK_EXCEPTIONS
+from bravado.response import BravadoResponseMetadata
+
+
+class FakeIncomingResponse(IncomingResponse):
```

Comment:
Small naming suggestion: could we use `IncomingResponseMock` like for the other testing classes?
From documentation side I think that would be super helpful to have an example of usage

Reply:
I'm not sure yet how users would take advantage of `IncomingResponseMock`, i.e. what the testing scenarios would be. I'll add something once that's more clear. :)